src/styles/card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+
+  &__product-image {
+    display: flex;
+    margin: 32px 19px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    margin-left: 16px;
+    margin-top: 40px;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin-left: 16px;
+    margin-top: 4px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-left: 16px;
+    margin-right: 16px;
+    margin-top: 24px;
+
+    &-title {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 18px;
+      color: #616070;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $main-text-color;
+    }
+  }
+
+  &__button {
+    background-color: $accent-color;
+    border: 1px solid $accent-color;
+    border-radius: 5px;
+    display: flex;
+    align-items: center;
+    height: 39px;
+    width: 165px;
+    margin: 16px auto 0;
+
+    &-link {
+      color: #fff;
+      margin: 0 auto;
+      font-weight: 700;
+      font-size: 14px;
+      line-height: 16px;
+      text-decoration: none;
+    }
+
+    &:hover {
+      background-color: #fff;
+    }
+
+    &:hover .card__button-link {
+      color: $accent-color;
```

Comment:
The variable `$accent-color` is used here as well. Ensure it is defined to prevent compilation errors.